### PR TITLE
ed: refresh/add pages

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/ed.md [hunk 1]
```diff
@@ -1,36 +1,33 @@
...
+> See also: `awk`, `sed`.
 > More information: <https://www.gnu.org/software/ed/manual/ed_manual.html>.
 
-- Start ed, editing an empty document (which can be saved as a new file in the current directory):
+- Start an interactive editor session with an empty document:
 
 `ed`
 
-- Start ed, editing an empty document, with `:` as a command prompt indicator:
+- Start an interactive editor session with an empty document and the specified prompt:
```
Comment (navarroaxel) [inline pages/common/ed.md:11]:
```suggestion
- Start an interactive editor session with an empty document and a specific prompt:
```

File: pages/common/ed.md [hunk 2]
```diff
@@ -1,36 +1,33 @@
...
 
-`ed -p :`
+`ed --prompt="> "`
 
-- Start ed editing an existing file (this shows the byte count of the loaded file):
+- Start an interactive editor session with user-friendly errors:
```
Comment (navarroaxel) [inline pages/common/ed.md:15]: The standard errors are hard to understand?

File: pages/osx/ed.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# ed
+
+> The original Unix text editor.
+> See also: `awk`, `sed`.
+> More information: <https://www.gnu.org/software/ed/manual/ed_manual.html>.
```
Comment (navarroaxel) [inline pages/osx/ed.md:5]: Do not support the same arguments in macOS?

**Review by blueskyson — APPROVED** [review]
Look good to me.

File: pages/common/ed.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply pages/common/ed.md:15]: According to `info ed` standard errors are very concise and may be hard to understand:

> Toggles the printing of error explanations. By default,
     explanations are not printed. It is recommended that ed scripts
     begin with this command to aid in debugging.

You can find this note in `H` command description (it is equivalent to `--verbose` option).

File: pages/osx/ed.md (hunk 3 above)
Comment (EmilySeville7cfg) [reply pages/osx/ed.md:5]: No, according to [this](https://www.freebsd.org/cgi/man.cgi?ed(1)) man page.

File: pages/common/ed.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages/common/ed.md:11]: I've used `a specific` everywhere.

**Review by marchersimon — APPROVED** [review]
Thanks!

**Comment by marchersimon** [comment]
@EmilySeville7cfg did you add the `stalebot ignore` label because the `CI / CI (push)` was failing? For some reason I can't even re-run it.

**Comment by EmilySeville7cfg** [comment]
Yes, I did but a time a go you removed this label.

**Comment by marchersimon** [comment]
Okay, just wanted to make sure. Any idea why the check is failing?

**Comment by EmilySeville7cfg** [comment]
I can watch tomorrow. 😉 Now I am going to sleep.

**Comment by EmilySeville7cfg** [comment]
@marchersimon, I am not sure but I think the problem was in that failed CI was outdated.

**Pull request merged**